app/adminPage.html

```diff
@@ -42,46 +42,51 @@ <h3 id="modal-title"> </h3>
 <section class="container">
     <div class="row">
         <div class="col-xs-12">
-            <h1>Welcome To The Admin Page</h1>
-                <h2>Add New User</h2>
+                <h2 class="second-heading">Add New User</h2>
                 <form class="container_controls" id="addNewUserForm">
                     <div class="container_input">
                         <input type="text" id="name" name="name" placeholder="Type new user's name">
                         <input type="text" id="email" name="email" placeholder="Type new user's email">
                         <input type="number" id="time" name="time" value="30">
                     </div>
                     <div class="container_submit">
-                        <input type="submit" value="Add User">
+                        <input type="submit" value="Add User" class="primary-button">
                     </div>
                 </form>
                 <div id='error' class='title_input'></div>
+                <h4 class="search-heading">Search User</h4>
+                    <input type="text" id="searchInput" placeholder="Type name or email">
+                    <input type="submit" id="searchSubmit" value="Search" class="primary-button">
+                    <input type="submit" id="searchReset" value="Clear Search" class="primary-button">
+                <div class="filterContainer">
+                    <h4 class="search-heading">Filter Results</h4>
+<!--                    <label for="filterScorePercentage" class="labelDropdown">Score percentage:</label>-->
```

Comment:
I know this is happening because you merged s1t5 branch. This should be reviewed when s1t5 branch's pull request is created.